stripe/res/layout/activity_add_source.xml

```diff
@@ -1,31 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
+<ScrollView
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     >
 
-    <com.stripe.android.view.CardMultilineWidget
-        android:id="@+id/add_source_card_entry_widget"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_margin="@dimen/add_card_total_margin"
-        />
-
-    <FrameLayout
-        android:id="@+id/add_source_error_container"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/add_source_card_entry_widget"
+        android:layout_height="match_parent"
         >
 
-        <TextView
-            android:id="@+id/tv_add_source_error"
+        <com.stripe.android.view.CardMultilineWidget
```

Comment:
nit: give this guy some guidance on whom to `layoutBelow` or `layout_alignParentTop` or whatever since he's in a relative layout now.